dev/docker/kerberos-hive/hdfs-site.xml

```diff
@@ -0,0 +1,88 @@
+<configuration>
+  <property>
+    <name>dfs.replication</name>
+    <value>1</value>
+  </property>
+
+  <property>
+    <name>dfs.webhdfs.enabled</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.permissions.superusergroup</name>
+    <value>hdfs</value>
+  </property>
+
+  <property>
+    <name>dfs.block.access.token.enable</name>
+    <value>true</value>
+  </property>
+
+  <property>
+    <name>dfs.namenode.kerberos.principal</name>
+    <value>hdfs/mockhost@HADOOPKRB</value>
+  </property>
+  <property>
+    <name>dfs.namenode.keytab.file</name>
+    <value>/hdfs.keytab</value>
+  </property>
+  <property>
+    <name>dfs.namenode.kerberos.internal.spnego.principal</name>
+    <value>HTTP/mockhost@HADOOPKRB</value>
+  </property>
+  <property>
+    <name>dfs.web.authentication.kerberos.keytab</name>
+    <value>/hdfs.keytab</value>
+  </property>
+  <property>
+    <name>dfs.web.authentication.kerberos.principal</name>
+    <value>HTTP/mockhost@HADOOPKRB</value>
+  </property>
+
+  <property>
+    <name>dfs.secondary.namenode.address</name>
+    <value>0.0.0.0:50088</value>
+  </property>
+  <property>
+    <name>dfs.secondary.namenode.kerberos.principal</name>
+    <value>hdfs/mockhost@HADOOPKRB</value>
+  </property>
+  <property>
+    <name>dfs.secondary.namenode.keytab.file</name>
+    <value>/hdfs.keytab</value>
+  </property>
+  <property>
+    <name>dfs.secondary.namenode.kerberos.internal.spnego.principal</name>
+    <value>HTTP/mockhost@HADOOPKRB</value>
+  </property>
+  <property>
+    <name>dfs.secondary.web.authentication.kerberos.keytab</name>
+    <value>/hdfs.keytab</value>
+  </property>
+  <property>
+    <name>dfs.secondary.web.authentication.kerberos.principal</name>
+    <value>HTTP/mockhost@HADOOPKRB</value>
+  </property>
```

Comment:
I think secondary namenode is not necessary. You don't need to configure and start it.